test/k8sT/hubble.go

```diff
@@ -198,6 +198,20 @@ var _ = Describe("K8sHubbleTest", func() {
 			res.ExpectContains(`hubble_flows_processed_total{protocol="TCP",subtype="to-endpoint",type="Trace",verdict="FORWARDED"}`)
 		})
 
+		It("Test TLS certificate", func() {
+			cmd := "cat /var/lib/cilium/tls/hubble/server.crt"
```

Comment:
I think we should limit the amount of shelling out that we do in tests, and prefer using native mechanisms, like in this case, using `ioutil.ReadFile`.

Reply:
In this case `cmd` is executed into the Cilium pod. I'm unsure how `ioutil.ReadFile` could be used to do that, but I would be happy to add a `ReadFile` in _test/helpers/wrappers.go_ if that is what you meant.

Reply:
:facepalm: You're right. The thought didn't even cross my mind that this is running inside a cluster. :sweat_smile: